### DISCUSSION TITLE
Rotation of movement plane, possible?

### DISCUSSION THREAD

**Discussion by nyaray** [linearmouse/linearmouse#481] [Q&A] [answered]
Hi, I was just wondering, is it possible to rotate the plane of motion, with the APIs that are exposed/in use in this app? The reason I'd like this is because I can grip my mouse much more comfortably when I hold it at a slight angle, which messes up my movement.

Thank you for a great piece of software, the acceleration removing is a must for me, so I'm really glad that this exists <3

**Comment by lujjjh** [comment, marked as answer]
Thank you for your idea! Based on my knowledge of existing APIs, I don't think it's feasible. It's likely that creating a mouse driver would be necessary in order to achieve this functionality.
